components/expenses/PayExpenseModal.js

```diff
@@ -123,6 +141,17 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, host, error }
   const hasAutomaticManualPicker = ![PayoutMethodType.OTHER, PayoutMethodType.ACCOUNT_BALANCE].includes(
     payoutMethodType,
   );
+  const canQuote = host.transferwise && payoutMethodType === PayoutMethodType.BANK_ACCOUNT;
+  const quoteQuery = useQuery(quoteExpenseQuery, {
+    variables: { id: expense.id },
+    context: API_V2_CONTEXT,
+    skip: !canQuote,
+  });
+
+  const paymentProcessorFee = quoteQuery?.data?.expense?.quote.paymentProcessorFeeAmount || {
+    valueInCents: formik.values.paymentProcessorFee,
+    currency: expense.currency,
+  };
```

Comment:
Won't this ignore the user input for payment processor fees when selecting "Manual" payment?

Reply:
Oh, nice catch. Let me check and fix this.